src/components/scene/stats.js

```diff
@@ -11,6 +11,11 @@ var AFrameStats = window.aframeStats;
  * Stats appended to document.body by RStats.
  */
 module.exports.Component = registerComponent('stats', {
+  schema: {
+    type: 'boolean',
```

Comment:
@dmarcos so that can be removed then also from `visible component` so just having
`schema: { default: true }` right?